### PR TITLE
US-18 ReviewVenue

### PR DESCRIPTION
**Pull request by toshydev**
- initiate MongoDB connection
- create db models
- implement VenueData in VenueDetail component
- create getRating helper function
- create ReviewList component
- create ReviewSection component
- implement ReviewList in VenueDetail
- fix prop passing
- fix date property in review document
- add ReviewForm and POST request handler
- implement create operations, update getRating function
- improve input elements
- add delete operation handling
- add update operation handling
- set review order by date

### CODE REVIEW

**Review by malgosiam — APPROVED** [review]
Awesome job! Your code is very clean and very readable! 🏄🏻‍♂️

**Comment by toshydev** [comment]
@malgosiam Thanks a lot 😁🙏

File: src/components/ReviewCard/index.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import { useState } from "react";
+import styled from "styled-components";
+import useSWR from "swr";
+import useSWRMutation from "swr/mutation";
+import DeleteButton from "../DeleteButton";
+import EditButton from "../EditButton";
+import Spinner from "../Spinner";
+
+const StyledReviewCard = styled.li`
+  background: #f0f0f0;
```
Comment (malgosiam) [inline src/components/ReviewCard/index.js:10]: Maybe this color could be added to your :root selector in the styles file

Comment (toshydev) [reply src/components/ReviewCard/index.js:10]: True! 💪There is a big styling/refactoring update coming soon. I just recently learned a few tricks from your last PR that I'm going to implement 🤩👌

Comment (malgosiam) [reply src/components/ReviewCard/index.js:10]: I am really pleased to read that ☺️

**Pull request merged**